pkg/cmd/roachtest/tests/admission_control_latency.go

```diff
@@ -245,7 +250,7 @@ func addMetamorphic(r registry.Registry, p perturbation, acceptableChange float6
 	// a given seed.
 	r.Add(registry.TestSpec{
```

Comment:
Probably a good idea to set the also set the `Randomized` field on this metamorphic test to opt it out of [test selection](https://cockroachlabs.atlassian.net/wiki/spaces/TE/pages/3596091522/Test+Selection+for+roachtest+nightlies), especially since we're doing this randomized cloud logic.

Reply:
Neat! I didn't realize that existed. Added.